### ISSUE TITLE
Does harbor have a plan to support replicate action which trigger by docker pull using pull mode?

### ISSUE THREAD

**Issue by yunkunrao** [goharbor/harbor#14736]
**Backgroud
Currently when one harbor cluster wants to sync with other harbor. In our case we create a replication rule with pull mode enabled. Admin had to sync image manually by clicking on replicate button. And it's a tedious task.

**Describe the solution you'd like**
When user executed cmd "docker pull XXX", and harbor didn't sync the image yet or the images is not latest, it will automatically pull from the other harbor. And it behaves much like proxy cache anyway.

**Comment by wy65701436** [comment]
Can you clarify the scenario? It seems that proxy cache can handle it?

**Comment by yunkunrao** [comment]
Yeah, currently proxy cache seems to be able to handle this scenario, but the prerequisite is docker client must change image name according. For example, original image name is demo-pro/redis:latest and user must add the prefix "proxy/" i.e. "proxy/demo-pro/redis:latest". In other words, we want to a solution which is transparent to docker client or k8s.
